roles/setup/tasks/main.yml

```diff
@@ -137,15 +138,15 @@
       changed_when: false
 
     - name: Check for hccm-usage-report Report
-      command: '{{ OCP_CLI }} get Report {{ setup_report_name }}'
+      command: '{{ OCP_CLI }} get Report {{ setup_usage_report_name }}'
```

Comment:
Is there more than one kind of report name that korekuta handles?

If there's only one kind of report, then this variable doesn't need to be renamed.

Reply:
Yes, there's usage and storage.